### PR TITLE
engineering: Wrap UIKit using Swift 5.4 @resultBuilder

### PR DESCRIPTION
**Pull request by viettrungphan**
The second part of Building UI with UIKit using Builder pattern (DSL style)

### CODE REVIEW

File: Engineering/Mobile/Swift 5.4 @resultBuilder.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+}
+```
+
+Thank you for following us to the end of this brainey, hopfully you enjoy it.
+
+If you have anyquestion feel free to email to support@dwarvesv.com.
+
+If you like it please give us a ⭐️ to bring more brainey in the futute.
+
+Happy coding 🧑‍💻!
```
Comment (monotykamary) [inline Engineering/Mobile/Swift 5.4 @resultBuilder.md:239]: Please avoid pleasantries in your notes. Brainery notes are close to fleeting notes, which are often semantically not aware of any subject or audience outside its context.

**Review by monotykamary — CHANGES_REQUESTED** [review]
Format seems OK, but please do avoid pleasantries and advertisements in these notes. Fleeting notes do not target an audience *outside of its context.

There is also a non-trivial amount of spelling errors, which we can help fix up a bit.

File: Engineering/Mobile/Swift 5.4 @resultBuilder.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+        UIButton()
+            .mintStyle()
+            .title("Update Now")
+            .tap(action: { [weak self] in
+                navigateToAppStore()
+            })
+            .heightAnchor(height: 44)
+    }
+}
+
```
[the remark this answers is not in the document]
Comment (viettrungphan) [reply Engineering/Mobile/Swift 5.4 @resultBuilder.md:179]: It's normal in Swift for readability.

**Review by zlatanpham — APPROVED** [review]
Note that I will do a light edit before we publish it.

**Pull request merged**